pkg/serverless/trace/trace.go

```diff
@@ -131,9 +147,44 @@ func filterSpanFromLambdaLibraryOrRuntime(span *pb.Span) bool {
 			return true
 		}
 	}
+
```

Comment:
Let's add comments above the http, tcp and dns blocks so there's a clear differentiator